### DISCUSSION TITLE
ccf-validation-tools/Thymus/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#288] [General]
# ccf-validation-tools/Thymus/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Thymus/comments/

**Comment by emquardokus** [comment]
Thymus_v1.4_DRAFT
https://docs.google.com/spreadsheets/d/1tbHMjOi7wPXnq3TFp74N2kPKtiJ_pFSgz5u5CZOGGrc/edit#gid=863370556

**Different labels**


    In row 31, the term CL:0009082 has different name/label in the source ontology. The name/label in the ASCT+B table is committed double negative thymocyte and the one in the ontology is committed double negative thymocyte (Homo sapiens). For reference, the given name/label by SMEs is double negative thymocyte 3. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

1.  FIXED: specified double negative thymocyte (Homo sapiens)   In row 30, the term CL:0009081 has different name/label in the source ontology. The name/label in the ASCT+B table is specified double negative thymocyte and the one in the ontology is specified double negative thymocyte (Homo sapiens). For reference, the given name/label by SMEs is double negative thymocyte 2. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2.  FIXED: immature NK T cell   In row 78, the term CL:0000914 has different name/label in the source ontology. The name/label in the ASCT+B table is immature NKT cell and the one in the ontology is immature NK T cell. For reference, the given name/label by SMEs is immature NKT cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3.  FIXED rearranging double negative thymocyte (Homo sapiens)   In row 32, the term CL:0009083 has different name/label in the source ontology. The name/label in the ASCT+B table is rearranging double negative thymocyte and the one in the ontology is rearranging double negative thymocyte (Homo sapiens). For reference, the given name/label by SMEs is double negative thymocyte 4. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Thymus_v1.4_DRAFT
https://docs.google.com/spreadsheets/d/1tbHMjOi7wPXnq3TFp74N2kPKtiJ_pFSgz5u5CZOGGrc/edit#gid=863370556

**Different labels**

FIXED:  The term CL:0009082 has a different name/label in the source ontology in the following 1 row 31. The name/label in the ASCT+B table is committed double negative thymocyte and the one in the ontology is committed double negative thymocyte (Homo sapiens). For reference, the given name/label by SMEs is double negative thymocyte 3. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Thymus_v1.4
https://docs.google.com/spreadsheets/d/1tbHMjOi7wPXnq3TFp74N2kPKtiJ_pFSgz5u5CZOGGrc/edit#gid=863370556

**Typos or punctuation mistakes**

The CL label was present in the CT/1/ID, this has been fixed: row 31--> CL:0009082
    It might have a typo in the term COMMITTED in the following 1 row 31. The term id should have this pattern: UBERON:NNNNNNN or CL:NNNNNNN or PCL:NNNNNNN. The ontology name in upper case. N is a number, and it should have exactly 7 numbers after the colon. Please change it in the ASCT+B table.

**Reply by anitacaron** [reply]
Hi Ellen, the label is back in the place of the CT/1/ID in the v1.5 https://docs.google.com/spreadsheets/d/1EPl6_WSbS2Kl6T7pKWKGt9UCHvOBnZ5RXm45IpMVLVA/edit#gid=863370556&range=X31:X31

**Comment by anitacaron** [comment]
Thymus_v1.5_DRAFT https://docs.google.com/spreadsheets/d/1EPl6_WSbS2Kl6T7pKWKGt9UCHvOBnZ5RXm45IpMVLVA/edit?pli=1#gid=863370556 

**FIXED - Add CL:0009082** 1. It might have a typo in the term COMMITTED in the following 1 row 31. The term id should have this pattern: UBERON:NNNNNNN or CL:NNNNNNN or PCL:NNNNNNN. The ontology name in upper case. N is a number, and it should have exactly 7 numbers after the colon. Please change it in the ASCT+B table.

**Comment by emquardokus** [comment]
Thymus_v1.4_DRAFT
https://docs.google.com/spreadsheets/d/1tbHMjOi7wPXnq3TFp74N2kPKtiJ_pFSgz5u5CZOGGrc/edit#gid=863370556

**Miscellaneous fixes**
1. Update all Ref/n/DOI to Ref/n/ID
2. Update all DOI references to resolvable format
